tests/modules/segmentation/filter/test_filter_z_score.py

```diff
@@ -0,0 +1,217 @@
+import pytest
+import h5py
+import numpy as np
+from itertools import product
+import pathlib
+import json
+import networkx
+
+from ophys_etl.modules.decrosstalk.ophys_plane import OphysROI
+
+from ophys_etl.modules.segmentation.utils.roi_utils import (
+    ophys_roi_to_extract_roi)
+
+from ophys_etl.modules.segmentation.graph_utils.conversion import (
+    graph_to_img)
+
+from ophys_etl.modules.segmentation.utils.roi_utils import (
+    background_mask_from_roi_list,
+    deserialize_extract_roi_list)
+
+from ophys_etl.modules.segmentation.filter.filter_utils import (
+    z_vs_background_from_roi)
+
+from ophys_etl.modules.segmentation.filter.roi_filter import (
+    ZvsBackgroundFilter)
+
+from ophys_etl.modules.segmentation.modules.filter_z_score import (
+    ZvsBackgroundFilterRunner)
+
+from ophys_etl.modules.segmentation.processing_log import (
+    SegmentationProcessingLog)
+
+
+@pytest.fixture(scope='session')
+def img_shape_fixture():
+    return (64, 64)
+
+
+@pytest.fixture(scope='session')
+def roi_list_fixture(img_shape_fixture):
+    rng = np.random.default_rng(771223)
+    roi_list = []
+    for ii in range(10):
+        x0 = rng.integers(0, img_shape_fixture[1]-5)
+        width = min(img_shape_fixture[1]-x0,
+                    rng.integers(5, 8))
+        y0 = rng.integers(0, img_shape_fixture[0]-5)
+        height = min(img_shape_fixture[0]-y0,
+                     rng.integers(5, 8))
+        mask = rng.integers(0, 2, (height, width)).astype(bool)
+        roi = OphysROI(x0=int(x0), width=int(width),
+                       y0=int(y0), height=int(height),
+                       valid_roi=True, roi_id=ii,
+                       mask_matrix=mask)
+        roi_list.append(roi)
+    return roi_list
+
+
+@pytest.fixture(scope='function')
+def processing_log_path_fixture(tmpdir, roi_list_fixture, seeder_fixture):
+    extract_roi_list = [ophys_roi_to_extract_roi(roi)
+                        for roi in roi_list_fixture]
+
+    log_path = pathlib.Path(tmpdir) / 'roi_log_for_filter_test.h5'
+    log = SegmentationProcessingLog(log_path, read_only=False)
+    log.log_detection(attribute='dummy_metric',
+                      rois=extract_roi_list,
+                      group_name='detect',
+                      seeder=seeder_fixture)
+    yield log_path
+
+
+@pytest.fixture(scope='session')
+def roi_list_path_fixture(tmpdir_factory, roi_list_fixture):
+    tmpdir_path = pathlib.Path(tmpdir_factory.mktemp('z_score_roi'))
+    roi_path = tmpdir_path / 'input_rois.json'
+    extract_roi_list = [ophys_roi_to_extract_roi(roi)
+                        for roi in roi_list_fixture]
+    with open(roi_path, 'w') as out_file:
+        out_file.write(json.dumps(extract_roi_list, indent=2))
+    yield roi_path
+
+
+@pytest.fixture(scope='session')
+def graph_fixture(tmpdir_factory, roi_list_fixture, img_shape_fixture):
```

Comment:
there was another graph fixture and others in a similar file.
Can they be moved up to `conftest.py`
If not, can you add some small docstrings to the fixtures to say what makes them special for these tests?